WebHostLib/static/assets/tutorial/v6/setup_en.md

```diff
@@ -0,0 +1,37 @@
+# VVVVVV MultiWorld Setup Guide
+
+## Required Software
+
+- VVVVVV ([Steam](https://store.steampowered.com/app/70300/VVVVVV/) or [GOG](https://www.gog.com/game/vvvvvv), NOT Make and Play Edition!)
```

Comment:
Please provide more context for the links here describing the page they are going to. This is for users who may not be equally sighted.

e.g.: instead of a hyperlink saying `Steam` is can be `VVVVVV Steam Store Page`.